pkg/capacity/capacity.go

```diff
@@ -99,3 +103,15 @@ func (r *ResourceOracle) Disks() (d Disks, err error) {
 
 	return
 }
+
+//GetHypervisor gets the name of the hypervisor used on the node
+func (r *ResourceOracle) GetHypervisor() (s []string, err error) {
+	out, err := exec.Command("virt-what").CombinedOutput()
+
+	if err != nil {
+		return s, fmt.Errorf("could not detect if VM or not: %s", string(out))
+	}
+
+	out = bytes.TrimSpace(out)
+	return s, nil
```

Comment:
```suggestion
      return strings.Split(string(out), "\n"), nil
```
you forgot to return the actual content of the command result, now you just return an empty slice of string